openrarity/providers/opensea/client.py

```diff
@@ -177,7 +210,29 @@ async def fetch_opensea_assets_data(
     before_sleep=before_sleep.before_sleep_log(logger, logging.WARN),
 )
 async def _send_request(client: "httpx.AsyncClient", url: str, params: dict[str, Any]):
+    """Sends httpx.asyncclient request to get token assests data.
```

Comment:
Deferred function to fetch params from a url and handle errors. Note that it returns a specific error for rate limits and non-200 responses.